docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,103 @@ which groups these values were observed, as well as the difference and ratio
 between the maximum and the minimum values. For more information refer to the
 subpackage :code:`fairlearn.metrics`.
 
+Failure modes encountered when abstracting
+--------------------------------------------------
+
+Machine learning systems used in the real world are inherently sociotechnical
+systems, which include both technologies and social actors. Designers of machine
+learning systems who fail to consider the way in which social contexts and technologies
+are interrelated are at risk of falling into "abstraction traps". In this section,
+we explain what those traps are, and give some suggestions on how you can avoid them.
+
+In "Fairness and Abstraction in Sociotechnical Systems" Selbst et al. [#4]_
+identify failure modes that arise from applying abstraction. They identify
+them as:
+
+* *The Solutionism Trap*
+
+* *The Ripple Effect Trap*
+
+* *The Formalism Trap*
+
+* *The Portability Trap*
+
+* *The Framing Trap*
+
+These traps are inspired by the observation that each of these traps are the
+result of failing to consider the way in with social context and technology
+are interrelated, as well as a deeper understanding of "the social" in order to
+solve problems Ackerman [#5]_.
+
+The Solutionism Trap
+^^^^^^^^^^^^^^^^^^^^
+
+This trap occurs when data scientists assume that the best solution to a problem involves
```

Comment:
I am a bit torn between (1) speaking to our primary audience (data scientists), (2) acknowledging that data scientists usually don't (and probably shouldn't) make these decisions in isolation (e.g., influenced by management, product owners, project managers, etc.)

WDYT @mmadaio @lgfunderburk?

Reply:
yeah, I caught myself thinking something along these lines
Data scientists are those who implement models, but data scientists are not necessarily in charge of collecting data or creating data collections for the model

Reply:
Open to options, can change into "we" or "one"
When one assumes that the best solution involves....
Would this work better @hildeweerts ?

Reply:
changed it to we